modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqTranslate.scala

```diff
@@ -70,10 +70,16 @@ class SeqTranslate(site: Site) {
         None,
         config.toStepConfig,
         false,
+        (if(i==0) List(List(toAction(systems.odb.sequenceStart(obsId, "").map(_ => Result.Ignored))))
```

Comment:
So, sequenceStart notification is sent at the beginning of the first step in the sequence. Here I know it is the first step, because its index is 0.
SequenceComplete must be sent at the end of the last step. I have the index, but to identify the last one here I need to know the total number of steps. Instead of passing that value, I directly pass a flag.